doc/architecture/decisions/0012-use-aws-for-hosting.md

```diff
@@ -0,0 +1,17 @@
+# 1. use-aws-for-hosting
+
+Date: 2022-04-07
+
+## Status
+
+Accepted
+
+## Context
+
+This ADR is to document a past decision retrospectively for reference. The context for this decision is now unknown.
```

Comment:
As I wasn't here when this was written I have no context to add. If someone has some words I'd be happy to add them in?

Reply:
Something like:

```
The Scenic Or Not Rails application is being introduced to replace an old, legacy PHP application, which is over ten years old. The team responsible for maintaining the application are comfortable with AWS, and are happy to manually deploy the application. This may be reviewed in the future.
```